.dockerignore

```diff
@@ -12,6 +12,6 @@ LICENSE
 .pylintrc
 .editorconfig
 env-docker
-templates/includes/base_icons.html
-static/build
+server/templates/includes/base_icons.html
+server/static/build
```

Comment:
<!-- Issues highlight specific problems with the subject under review. -->
**issue:** I think this also needs to be updated in `.gitignore`.

I'm getting a lot of untracked build files.